src/cdk/portal/portal.md

```diff
@@ -55,7 +55,43 @@ portals, it must be included in the `entryComponents` of its `NgModule`.
 
 Usage:
 ```ts
-this.userSettingsPortal = new ComponentPortal(UserSettingsComponent);
+ngAfterViewInit() {
+  this.userSettingsPortal = new ComponentPortal(UserSettingsComponent);
+}
+```
+
+##### `TemplatePortal`
+Used to create a portal from an embedded template.
+
+Usage:
+```html
+<ng-template #templatePortalContent>Some content here</ng-template>
+```
+
+```ts
+@ViewChild('templatePortalContent') templatePortalContent: TemplateRef<any>;
```

Comment:
```suggestion
@ViewChild('templatePortalContent') templatePortalContent: TemplateRef<unknown>;
```

I'd want to use `unknown` here instead of `any` just to avoid showing examples that use `any` in general